packages/core/src/tools/tool-builder/builder.test.ts

```diff
@@ -139,11 +139,12 @@ async function runSingleOutputsTest(
 
     const generateOptions: any = {
       maxSteps: 1,
-      structuredOutput: {
-        schema: testTool.inputSchema!,
-        model: model,
-        errorStrategy: 'strict',
-      },
```

Comment:
I think this might'vebeen here intentionally since the `structuredOutput` key uses the new output processor internally